examples/tests/sdk_tests/counter/inc.test.w

```diff
@@ -3,32 +3,32 @@ bring cloud;
 let counter = new cloud.Counter(initial: 0);
```

Comment:
Can you please remove the comments here?
you can add a new inc test (you can choose the name)
for incrementing two kinds of counters- one without initial and one with initial != 0
please make sure to pass through current value of -1 -> 0 -> 1 too (to avoid the problem in the other comment)